Game.java

```diff
@@ -0,0 +1,167 @@
+
+public class Game {
+    private int[][] board;
+    private int currentPlayer;
+    private boolean isFinished;
+
+    public int[][] getBoard() {
+        return board;
+    }
+
+    public int getCurrentPlayer(){
+        return currentPlayer;
+    }
+
+    public Game() {
```

Comment:
consider getting the board size as input

Reply:
But in four in a row its always the same!!!!